Baseball/Team Building.user.js

```diff
@@ -81,7 +82,7 @@ async function IncreasePlayer(id, type, playerID, verify) {
 				.filter(inputTag => inputTag.value == playerID)
 				.slice(1)
 				.map(inputTag => inputTag.parentElement)
-				.find(formTag => formTag.children[1].name === type).textContent.trim());
+				.find(formTag => formTag.children[buttonId].name === type).textContent.trim());
```

Comment:
`buttonId` can either be zero or one, but when one, will throw an error as it will be looking at the stop buttons for the 20k upgrades and not the up buttons for the 2k. I imagine a simple fix would be multiplying it by two.

Reply:
Nvm about this. I'm mistaken. It's working on a raw document and not a modified one so it being 1 instead of 2 works just fine.